### PR TITLE
edit pass: cmake-presets

### PR DESCRIPTION
**Pull request by ShawnJackson**
@TylerMSFT, @esweet431 – Please review the copy edit to this article and let me know if you have any feedback. This is for Azure DevOps work item [1840349](https://mseng.visualstudio.com/TechnicalContent/_workitems/edit/1840349).

### CODE REVIEW

File: docs/cmake-presets.md [hunk 1]
```diff
@@ -25,29 +25,30 @@ A new setting, `cmake.useCMakePresets`, has been added to `settings.json`:
...
 
 ### CMake: Select Build Preset
 
-**CMake: Select Build Preset** lists the Default Build Preset and the union of non-hidden Build Presets defined in `CMakePresets.json` and `CMakeUserPresets.json`. The Default Build Preset is equivalent to passing `cmake --build` with no additional arguments from the command line. Select a Build Preset to make it the active Build Preset. This is the `buildPreset` used when CMake is invoked to build the project. The active Build Preset is displayed in the status bar.
+**CMake: Select Build Preset** lists the default Build Preset and the union of non-hidden Build Presets defined in `CMakePresets.json` and `CMakeUserPresets.json`. The default Build Preset is equivalent to passing `cmake --build` with no additional arguments from the command line. Select a Build Preset to make it the active Build Preset. This is the `buildPreset` value that's used when CMake is invoked to build the project. The active Build Preset is displayed on the status bar.
```
Comment (TylerMSFT) [inline docs/cmake-presets.md:51]: I'm not sure on the casing for default Build Preset. @esweet431, is Default Build Preset how the CMake docs refer to this? In other words, is 'Default Build Preset' a thing, or are we talking about the default `Build Preset`?

**Review by TylerMSFT — APPROVED** [review]
I left a question for Erika about capitalization in one place, but as I don't have any other concerns, I'm setting 'approved' for my part.

File: docs/cmake-presets.md (hunk 1 above)
Comment (esweet431) [reply docs/cmake-presets.md:51]: The "Default" Build Preset is a VS-specific concept. It's name is "Default" (that's how it shows up to users) so I used the same casing for the docs.

Comment (ShawnJackson) [reply docs/cmake-presets.md]: Understood. I just reverted the change.

**Comment by ShawnJackson** [comment]
@TylerMSFT, @esweet431 – I addressed the feedback. Is this one ready for sign-off?

**Review by TylerMSFT — APPROVED** [review]
LGTM - thank you, Shawn.

**Comment by ShawnJackson** [comment]
#sign-off

**Comment by ShawnJackson** [comment]
@TylerMSFT - Even though you approved this PR, merging is blocked. Do you know who can give approval to allow merging?

**Comment by TylerMSFT** [comment]
I think @bobbrow is the one who can merge this. Or possibly @esweet431

**Review by bobbrow — APPROVED** [review]

**Pull request merged**